home/containers/Project.tsx

```diff
@@ -0,0 +1,109 @@
+import { useQuery } from '@apollo/client';
+import { StackScreenProps } from '@react-navigation/stack';
+import gql from 'graphql-tag';
+import { AllStackRoutes } from 'navigation/Navigation.types';
+import * as React from 'react';
+
+import ProjectView from '../components/ProjectView';
+
+export interface ProjectUpdateUpdate {
+  id: string;
+  group: string;
+  message: string;
+  createdAt: Date;
+  runtimeVersion: string;
+  platform: 'android' | 'ios';
+}
+
+export interface ProjectUpdateBranch {
+  id: string;
+  name: string;
+  updates: ProjectUpdateUpdate[];
+}
+
+export interface ProjectUpdateChannel {
+  id: string;
+  name: string;
+  updateBranches: ProjectUpdateBranch[];
+}
+
+export type ProjectDataProject = {
+  id: string;
+  name: string;
+  slug: string;
+  fullName: string;
+  username: string;
+  published: string;
+  description: string;
+  githubUrl?: string;
+  playStoreUrl?: string;
+  appStoreUrl?: string;
+  sdkVersion: string;
+  iconUrl?: string;
+  privacy: string;
+  icon?: {
+    url: string;
+    primaryColor?: string;
+    colorPalette?: object;
+  };
+  updateChannels: ProjectUpdateChannel[];
+};
+
+export interface ProjectData {
+  app: { byId: ProjectDataProject };
+}
+
+interface ProjectVars {
+  appId: string;
+}
+
+export const ProjectPageQuery = gql`
+  query WebContainerProjectPage_Query($appId: String!) {
+    app {
+      byId(appId: $appId) {
+        id
+        name
+        slug
+        fullName
+        username
+        published
+        description
+        githubUrl
+        playStoreUrl
+        appStoreUrl
+        sdkVersion
+        iconUrl
+        privacy
+        icon {
+          url
+        }
+        updateChannels(limit: 10000, offset: 0) {
```

Comment:
A limit of 10000 is going to take a very long time to download -- something like 100 seems reasonable. Also we should figure out how we're going to query by runtime version instead of having to filter locally.

Reply:
I think we'll actually want to show a paginated page of all versions (not just those with matching runtime versions) and then tell the user in the UI which ones can be opened in Expo Go and which ones must be opened in a dev client. That way there's no wondering why an update isn't showing up in this interface. The exact UI has yet to be determined though.

As for the 10000, this is the same query that is used in eas-cli. I'll set it to 100 for now since it doesn't much matter in the interim MVP state but as I mentioned above I think we'll probably do a paginated UI for these in the near future. I just didn't want to built it ahead of time since it's a bit of work.